CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## [v1.0.0] - 2022-04-18
+
+Inclusion of kyverno Policies in CSM 1.2.5 Patch
```

Comment:
Change this to not include the CSM release. Just note that it is the initial version, what is actually in it.

Reply:
okay